src/constants.ts

```diff
@@ -12,3 +14,14 @@ export const SUPPORTED_EXTENSION_IDS = [
 ];
 
 export const SIMPLE_DECISIONS_ACTION_CODE = '0x12345678';
+
+// These are indexes of data slots in contracts
+// For more information: https://github.com/JoinColony/colonyNetwork/blob/develop/contracts/colony/ColonyStorage.sol
+export const EXPENDITURES_SLOT = BigNumber.from(25);
+export const EXPENDITURESLOTS_SLOT = BigNumber.from(26);
+
+export const EXPENDITURE_OWNER_AND_STATUS = toB32(BigNumber.from(0));
+
+export const EXPENDITURESLOT_RECIPIENT = toB32(BigNumber.from(0));
+export const EXPENDITURESLOT_CLAIMDELAY = toB32(BigNumber.from(1));
+export const EXPENDITURESLOT_PAYOUTMODIFIER = toB32(BigNumber.from(2));
```

Comment:
This is very confusing to read, as we're mixing "magic" numbers (slot indexes)  with actual numbers 

I'd change this to something like:
```js
export const EXPENDITURES_SLOT = BigNumber.from(SlotIndex.Expenditures);
export const EXPENDITURESLOTS_SLOT = BigNumber.from(SlotIndex.ExpenditureSlots);
```

_(Naming sucks because of they network named these)_